packages/rancher-windows-gmsa/charts/questions.yaml

```diff
@@ -0,0 +1,59 @@
+questions:
+  - variable: credential.enabled
+    default: true
+    description: Create a GMSA Credential Spec when installing GMSA Webhook
+    label: Create a GMSA Credential Spec
+    type: boolean
+    group: "Credential Spec"
+    show_subquestion_if: true
+    subquestions:
+      - variable: credential.domainJoinConfig.machineAccountName
+        label: GMSA Account Name
+        description: Username of the GMSA account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.guid
+        label: GUID
+        description: GUID of the Service Account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.sid
+        label: SID
+        description: SID of the GMSA Account
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.dnsName
+        label: DNS Domain Name
+        description: Name of the domain in DNS
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.dnsTreeName
+        label: DNS Tree Domain
+        description: Root name of the domain in DNS
+        type: string
+        required: true
+      - variable: credential.domainJoinConfig.netBiosName
+        label: NETBIOS Name
+        description: NETBIOS Name for the domain.
+        type: string
+        required: true
+
+  - variable: certificates.certManager.enabled
+    default: true
+    description: Enable cert manager integration. Cert manager should be already installed

```

Comment:
```suggestion
    description: Use cert-manager to generate certificates for the webhook
```

Reply:
this wasn't resolved or addressed? If you like the original wording better, we should still change `cert manager` to `cert-manager` to keep the style consistent with what upstream uses in https://cert-manager.io/

Reply:
I missed that.